lib/pharos_cluster.rb

```diff
@@ -8,11 +8,9 @@
 require_relative "pharos/root_command"
 
 module Pharos
-  CRIO_VERSION = '1.10.4'
-  CRICTL_VERSION = '1.0.0-beta.0'
```

Comment:
Removed `CRICTL_VERSION` and `DOCKER_VERSION` because they are not used anymore.